plugins/polio/js/src/hooks/useSaveCampaign.js

```diff
@@ -8,6 +10,71 @@ const formatGroupedCampaigns = groupedCampaigns => {
         return commaSeparatedIdsToStringArray(groupedCampaigns);
     return groupedCampaigns ?? [];
 };
+
+const convertFormattedNumbers = body => {
```

Comment:
Arg this is a bit ugly

Reply:
Yeah, but I don't see how else to do it without refactoring the whole thing, and I don't fancy doing that for putting commas in a number